<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UartChoise</class>
 <widget class="QWidget" name="UartChoise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QComboBox" name="speedChose">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>B9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B921600</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="serial_find">
      <property name="text">
       <string>Serial find</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="uartFind">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Seleziona la Porta Seriale&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Uart Speed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UartChoise</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
